libphal/phal_exception.H

```diff
@@ -125,7 +130,7 @@ class SbeError final : public phalError
 	int getFd() const noexcept
 	{
 		if (!ffdcFileList.empty()) {
-			const auto& tuple = ffdcFileList.begin()->second;
+			const auto &tuple = ffdcFileList.begin()->second;
```

Comment:
I guess you have had run clang format as well with your changes concurrently. Make two commits separately, one for your changes and other for clang formatting, for clear understanding.